components/tooltip/test/tooltip.test.js

```diff
@@ -32,12 +32,29 @@ describe('d2l-tooltip', () => {
 		it('should add aria-labelledby to its target if for-type is \'label\'', async() => {
 			const tooltipLabelFixture = await fixture(labelFixture);
 			const target = tooltipLabelFixture.querySelector('#label-target');
-			expect(target.hasAttribute('aria-labelledby')).to.be.true;
+			const tooltip = tooltipLabelFixture.querySelector('d2l-tooltip');
+			expect(target.getAttribute('aria-labelledby')).to.equal(tooltip.id);
```

Comment:
Not really part of this change, but figured it was better to check that the value it set was actually... correct.